.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+### Description
+
+For a bug: Describe the bug and list the steps you used when the issue occurred.
+include any files necessary to reproduce this by zipping them and attaching them
+to this issue. For features, describe what you'd like, how it might be used and
+why.
+
+#### Steps to reproduce | Feature Idea (Delete one)
+  1. Click on this
+  2. Do that
+  3. ...
+
+For an enhancement or new feature: Describe your needs or wants.
+
+### Version effected (if bug)
+See contributing documentation linked above if you can't find the version.
+
+#### What operating system are you running?
+Windows: 7, 8, 10 ?
+OS X: 10.?
+Linux: Distribution (Ubuntu, Debian, RedHat, Fedora) and version
+
+#### How did you install RoboPaint?
+Running from source
+System installer
+Copied files from zip
+
+#### What robot are you using?
```

Comment:
Probably, though if someone has gone to the trouble of doing this, it'll be clear to them that it's different. Also it changes the version text.